bundle/default-bundle/start.sh

```diff
@@ -16,4 +16,4 @@ if [[ -n ${DEBUG_JVM_PRINT_JAVA_OPTS} ]]; then
   echo "Applied JVM options: $JAVA_OPTS"
 fi
 
-exec java ${JAVA_OPTS} -cp '/opt/app/*' "io.camunda.connector.runtime.app.ConnectorRuntimeApplication"
+exec java ${JAVA_OPTS} -cp '/opt/app/*:/opt/additionalLibs/*' "io.camunda.connector.runtime.app.ConnectorRuntimeApplication"
```

Comment:
```suggestion
exec java ${JAVA_OPTS} -cp '/opt/app/*;/opt/additionalLibs/*' "io.camunda.connector.runtime.app.ConnectorRuntimeApplication"
```

AFAIK you have to use a semicolon here: https://stackoverflow.com/questions/58507668/set-java-classpath-to-multiple-folders-that-have-jar-files-and-run-from-command